cloud-data-protection/src/components/logout.tsx

```diff
@@ -0,0 +1,26 @@
+import React, {useEffect} from "react";
+import {AuthService} from "../services/authService";
+import {useSnackbar} from 'notistack';
+
+const Logout = () => {
+    const authService = new AuthService();
+
+    const { enqueueSnackbar } = useSnackbar();
+
+    const handleLogout = () => {
+        authService.logout();
+
+        enqueueSnackbar('You are now logged out', { autoHideDuration: 6000 })
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/olivierbouchomsfontys/cloud-data-protection/pullRequest?prid=7219402)
```suggestion
        enqueueSnackbar("You are now logged out", { autoHideDuration: 6000 })
```